src/components/mainComponent/mainComponent.js

```diff
@@ -12,12 +13,20 @@ class Main extends Component {
           <br />
           <h2>Welcome to React Tic Tac Toe</h2>
           <br/>
-          <div className="start_game">
-            <a className="human-page" href="./game/HumanVsHuman"><span><div id="start">Human vs. Human</div></span></a>
-          </div>
-          <div className="start_game">
-            <a className="computer-page" href="./game/ComputerVsHuman"><span><div id="start">Computer vs. Human</div></span></a>
-          </div>
+          <ButtonComponent
+            divClassName="start_game"
```

Comment:
It is a bit strange that your class names use both snake_case and kebab-case. I would stick to kebab-case.